app/assets/javascripts/test/backend-snapshot-tests/timetracking.e2e.js

```diff
@@ -45,6 +45,17 @@ test("getTimeTrackingForUser", async t => {
   t.snapshot(timeTrackingForUser, { id: "timetracking-timeTrackingForUser" });
 });
 
+test("getTimeTrackingForUser for another than the active user", async t => {
```

Comment:
for another **what**? user?

Reply:
Another user. From my understanding this is short for "for a user other than the active user" :)